### PR TITLE
a few small fixes

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/baggage/pull/61.html" title="Last updated on Jun 22, 2021, 4:11 AM UTC (26b97aa)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/baggage/61/d83bcbe...26b97aa.html" title="Last updated on Jun 22, 2021, 4:11 AM UTC (26b97aa)">Diff</a>

### CODE REVIEW

File: baggage/HTTP_HEADER_FORMAT.md [hunk 1]
```diff
@@ -5,21 +5,18 @@ A received header MAY be altered to change or add information and it SHOULD be p
 
 Multiple `baggage` headers are allowed. Values can be combined in a single header according to [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
 
-*See [rationale document](HTTP_HEADER_FORMAT_RATIONALE.md) for details of decisions made for this format.*
```
Comment (SergeyKanzhelev) [inline baggage/HTTP_HEADER_FORMAT.md:8]: rationale doc is not normative. We have a link to rationale from the readme

File: baggage/HTTP_HEADER_FORMAT.md [hunk 2]
```diff
@@ -5,21 +5,18 @@ A received header MAY be altered to change or add information and it SHOULD be p
...
-
-
-# Header Name
+## Header Name
```
Comment (SergeyKanzhelev) [inline baggage/HTTP_HEADER_FORMAT.md:8]: indent wasn't correct

File: baggage/privacy.md [hunk 3]
```diff
@@ -7,7 +7,8 @@ This standard makes it possible for systems to operate on a known, standardized
 Systems MUST assess the risk of header abuse. This section provides some considerations and initial assessment of the risk associated with storing and propagating this header. Systems may choose to inspect and remove sensitive information from the fields before processing or propagating the received data. All mutations should, however, conform to the list of mutations defined in this specification.
 
 ## Privacy of the baggage header
+
 The main purpose of this header is to provide additional system-specific information to other systems within the same trust-boundary.
-The `baggage` header may contain any <a href="#opaque">opaque</a> value in any of the keys.
+The `baggage` header may contain any <a>opaque</a> value in any of the keys.
```
Comment (SergeyKanzhelev) [inline baggage/privacy.md:12]: the link is actually `#dfn-opaque`. But without any `href` it works as well based on the definition word.

File: index.html [hunk 4]
```diff
@@ -34,9 +34,8 @@
       shortName: "baggage",
       format: "markdown",
       subjectPrefix: "baggage",
-      wg: "Distributed Tracing Working Group",
+      group: "wg/distributed-tracing",
```
Comment (SergeyKanzhelev) [inline index.html:37]: warning in a new version of respec. Transitioned to the new attribute

File: index.html [hunk 5]
```diff
@@ -57,9 +55,9 @@
   <section id='conformance'></section>
 
   <section data-include="abstract/baggage-overview.md" data-include-format='markdown'></section>
-  <section data-include="baggage/privacy.md" data-include-format='markdown'></section>
 
   <section data-include="baggage/HTTP_HEADER_FORMAT.md" data-include-format='markdown'></section>
+  <section data-include="baggage/privacy.md" data-include-format='markdown'></section>
```
Comment (SergeyKanzhelev) [inline index.html:60]: this ordering is more natural

File: w3c.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "group":     108594
-, "contacts":  ["caribouW3"]
+, "contacts":  ["plehegar"]
```
Comment (SergeyKanzhelev) [inline w3c.json:3]: @plehegar, this is ok, correct?

**Review by yurishkuro — APPROVED** [review]

**Review by danielkhan — APPROVED** [review]

**Review by plehegar — APPROVED** [review]

**Pull request merged**